### PR TITLE
Conda testing environment

### PR DESCRIPTION
**Pull request by onalant**
Based on requirements_dev.txt

@asford

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name: privileged_residues
```
Comment (asford) [inline environment.yml]: Is the name needed here w/ your `devenv.sh` script? Maybe better to remove if possible.

File: tests/test_networks.py [hunk 2]
```diff
@@ -11,6 +11,8 @@
 
 from privileged_residues import chemical
 
+pytestmark = pytest.mark.skip()
```
[the remark this answers is not in the document]
Comment (onalant) [reply tests/test_networks.py]: This skips the entire file. Would you prefer skipping the method?

File: tests/test_geometry.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+        ray = geometry.create_ray(center, base)
+
+        direction = center - base
+
+        assert_allclose(np.cross(direction, ray[:-1, 1]), 0.0)
+        assert_allclose(ray[:-1, 0], center)
+
+    @pytest.mark.filterwarnings("ignore:RuntimeWarning")
+    @pytest.mark.xfail # (strict=True)
```
[the remark this answers is not in the document]
Comment (onalant) [reply tests/test_geometry.py]: The failures are known edge-cases where floating-point inaccuracy makes the vector normalization check fail. I will add the comment now.

File: environment.yml (hunk 1 above)
Comment (onalant) [reply environment.yml]: No, it is not required. `devenv.sh` uses `conda env ... -p ./conda` which overrides the name in `environment.yml`. Will fix.
